app/models/component.rb

```diff
@@ -7,9 +7,10 @@ class Component < Aggregate
            -> { where(usage: 'signing', enabled: true) }, class_name: 'Certificate'
   has_many :disabled_signing_certificates,
            -> { where(usage: 'signing', enabled: false) }, class_name: 'Certificate'
-  has_one :encryption_certificate,
-          -> { where(usage: 'encryption').order(id: 'DESC') },
-          class_name: 'Certificate'
+
+  belongs_to :encryption_certificate, -> { where(usage: 'encryption') },
```

Comment:
can we extract the string `'signing'` and `'encryption'` to a global constants? So we can use a single variable instead of always typing a string (which is prone to errors/typos)